sdcm/utils/csrangehistogram.py

```diff
@@ -51,7 +51,7 @@ class HistorgramSummaryBase:
 
 class CSHistogram(HdrHistogram):
     LOWEST = 1
-    HIGHEST = 12 * 3600 * 1000 * 1000
+    HIGHEST = 24 * 3600_1000_1000_1000
```

Comment:
although this is an integer, i'm not sure it is doing exactly what you expected:
```
>>> 3600_1000_1000_1000
3600100010001000
```

did you mean to use 
```suggestion
    HIGHEST = 24 * 3600_000_000_000
```

?

Reply:
fixed